packages/components/package.json

```diff
@@ -1,35 +1,40 @@
 {
   "name": "@terra-ui-packages/components",
-  "version": "0.0.2",
-  "private": false,
```

Comment:
`yarn` automatically removed the "private" field, I assume because false is the default.
https://docs.npmjs.com/cli/v9/configuring-npm/package-json#private